tooling/babel-plugin-inject-package-version.js

```diff
@@ -2,20 +2,9 @@
  * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
  */
 
-const pkgUp = require('pkg-up');
 const t = require('babel-types');
 
-/**
- * Uses pkgUp to find the appropriate package.json for the specified babel state
- * object
- * @param {Object} state
- * @private
- * @returns {string}
- */
-function versionFromState(state) {
-  // eslint-disable-next-line global-require
-  return require(pkgUp.sync(state.file.opts.filename)).version;
-}
+const {version} = require('../package.json');
```

Comment:
When in the flow does this script get called?

Reply:
addressed in prerelease question